src/main/kotlin/app/FactCodes.kt

```diff
@@ -1,10 +1,19 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
 package app
 
 object FactCodes {
     val COMMITS_DAY_WEEK = 1
     val COMMITS_DAY_TIME = 2
+    val COMMITS_LINE_NUM_AVG = 8
+    val COMMITS_NUM = 9
+    // A map of line numbers to commits number. Used in a commit histogram.
+    val COMMITS_NUM_TO_LINE_NUM = 12
     val LINE_LONGEVITY = 3
     val LINE_LONGEVITY_REPO = 4
+    val LINE_LEN_AVG = 10
+    val LINE_NUM = 11
```

Comment:
it'd be really nice to document all of them.

Reply:
Agree